bundles/org.eclipse.equinox.api/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.eclipse.equinox.api
+Bundle-Version: 1.0.0.qualifier
+Export-Package: org.eclipse.core.runtime;version="3.7.0"
```

Comment:
Yes another bundle export this package split package.   

![image](https://github.com/user-attachments/assets/fc85b48c-9ed6-48db-b128-be437141acb0)

I assume some attributes will be needed.  I'm not sure if other importers will need to change?

Reply:
> I assume some attributes will be needed. I'm not sure if other importers will need to change?

No the goal is to have one package without mandatory attributes so users that want to reference the new api exported package do not need to specify anything and old users (that use an attribute) get the one as before.

Actually we then can replace all these special imports with regular ones.